records_mover/db/vertica/export_sql.py

```diff
@@ -21,7 +21,7 @@ def vertica_export_sql(db_engine: Engine,
     """
 
     def quote(value: str) -> str:
-        return quote_value(db_engine, value)
+        return quote_value(None, value, db_engine=db_engine)
```

Comment:
im assuming these None values are the old `db`

Reply:
yep!